src/styles.css

```diff
@@ -0,0 +1,177 @@
+@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro&display=swap");
+.App {
+  font-family: sans-serif;
+}
+
+:root {
+  --numbers-color: rgb(86, 142, 247);
+  --hands-color: rgb(86, 142, 247);
+  --timeSetTo-color:rgb(219, 139, 163);
+  --number-font-size: 22px;
+}
+
+.container{
+  margin: auto;
+  max-width: 600px;
+  width: 600vw;
+  max-height: 400px;
+  height: 400vh;
```

Comment:
Don't go over 100 in `vh` and `vw`, it's really bad practice. Setting a huge size and defining absolute max sized is the same as setting an absolute size. 
If you're having a hard time understanding what's `vh` and `vw`, think about it as that:
`vh` is % of the screen's height.
`vw` is % of the screen's width.
% shouldn't go over 100, so does `vh` and `vw`.
Here's a [guide I recommend](https://www.freecodecamp.org/news/css-unit-guide/)